components/ProfileElements/HelpModal.jsx

```diff
@@ -0,0 +1,92 @@
+import { View, Text } from "react-native";
+import { useState, useContext } from "react";
+import { set, ref } from "firebase/database";
+import { firebase_auth, firebase_db } from "../../firebase.config.js";
+import DropDownPicker from "react-native-dropdown-picker";
+import SmallFormButton from "../Form/SmallFormButton.jsx";
+import FormField from "../../components/Form/FormField";
+import { AlertContext } from "../../components/Alert/AlertProvider.jsx";
+import { ModalContext } from "../../components/Modal/ModalProvider.jsx";
+import { icon } from "@fortawesome/fontawesome-svg-core/import.macro";
+
+const furtherInfo = {
+    ReportError:
+        "Describe what happened in detail. If you can, include the steps you took to get to this point.",
+    AskQuestion: "Further details about your question.",
+    SuggestImprovement: "Further details about your suggestion.",
+};
+
+export default function HelpModal(props) {
+    const [type, setType] = useState("");
+    const [title, setTitle] = useState("");
+    const [description, setDescription] = useState("");
+    const [open, setOpen] = useState(false);
+    const [items, setItems] = useState([
+        { label: "Report an error", value: "ReportError" },
+        { label: "Ask a question", value: "AskQuestion" },
+        { label: "Suggest an improvement", value: "SuggestImprovement" },
+    ]);
+    const auth = firebase_auth;
+    const db = firebase_db;
+    const { setAlert } = useContext(AlertContext);
+    const { setModal } = useContext(ModalContext);
+
+    const report = () => {
+        if (title === "") {
+            alert("Please enter a title");
+            return;
+        }
+
+        try {
+            set(ref(db, "reports/" + Date.now()), {
+                uid: auth.currentUser.uid,
+                type: type ? type : "none",
+                title: title,
+                description: description ? description : "none",
+            });
+            setAlert("Report submitted!", "bg-green-500", icon({ name: "circle-check" }));
+            setModal(null);
+        } catch (error) {
+            setAlert("Error sending report", "bg-red-500", icon({ name: "circle-exclamation" }));
+        }
+    };
+
+    return (
+        <View className="flex flex-col h-full items-center">
+            <Text className="text-2xl font-extrabold text-center mb-1">Help</Text>
+            <View className="flex-1 flex-col m-4">
+                <DropDownPicker
```

Comment:
Don't know how easy this is, but it looks kinda weird with the different border widths when the dropdown is there?